config/locales/en.yml

```diff
@@ -131,7 +131,7 @@ en:
     qualification_details: Qualification details
     edit_other_qualification: Edit qualification
     destroy_other_qualification: Are you sure you want to delete this qualification?
-    becoming_a_teacher: Why do you want to teach
+    becoming_a_teacher: Why you want to teach
```

Comment:
@davidgisbey should we relabel this key to something like `personal_statement`? I suspect that `becoming_a_teacher` is a legacy from an old design?